### PR TITLE
Redesign the schedule.

### PR DESCRIPTION
**Pull request by mkuzak**

### CODE REVIEW

File: _includes/ror/schedule.html [hunk 1]
```diff
@@ -17,10 +17,11 @@ <h3>Day 1</h3>
...
     <table class="table table-striped">
       <tr> <td>09:30</td>  <td>Collaboration on GitHub </td> </tr>
-      <tr> <td>10:30</td>  <td>Publishing on Zenodo</td> </tr>
-      <tr> <td>11:00</td>  <td>Improve reusability with myBinder</td> </tr>
+      <tr> <td>10:30</td>  <td>Using Notebooks with version control</td> </tr>
+      <tr> <td>10:45</td>  <td>Publishing on Zenodo</td> </tr>
+      <tr> <td>11:15</td>  <td>Improve reusability with myBinder</td> </tr>
       <tr> <td>12:00</td>  <td>Lunch break</td> </tr>
-      <tr> <td>13:00</td>  <td>Scaling up by running Notebooks on SURFsara cluster.</td> </tr>
+      <tr> <td>13:00</td>  <td>Jypyter Notebooks continued</td> </tr>
```
Comment (c-martinez) [inline _includes/ror/schedule.html]: Is this where you would include widgets? How about 'Interactive notebooks' ? We could even include this extension: https://github.com/oschuett/appmode

Comment (mkuzak) [reply _includes/ror/schedule.html]: yes, among other things. I think first part of notebook will be mostly how to work with notebooks and markdown. The second part load some data, visualise, add interactive stuff.
appmode is a cool idea

Comment (c-martinez) [reply _includes/ror/schedule.html]: Perhaps 'Notebooks for data visualisation' or 'Notebooks for interacting with data' might be a more meaningful title?

Comment (mkuzak) [reply _includes/ror/schedule.html]: "Exploring data with Notebooks"?

Comment (c-martinez) [reply _includes/ror/schedule.html]: Yeah, that would work :-)

**Comment by mkuzak** [comment]
updated the schedule with new tittle for second Notebook session

**Pull request merged**